mscore/tourhandler.cpp

```diff
@@ -362,27 +362,31 @@ void TourHandler::positionMessage(QList<QWidget*> widgets, QMessageBox* mbox)
       // Calculate the topLeft point for the mbox
       QPoint displayPoint(0, 0);
       if (topBottom) {
-            bool displayAbove = (widgetsBox.center().y() > midY);
+            bool displayAbove = (widgetsBox.center().y() > midY);            
+            int mBoxHeight = mbox->size().height() + 15; // hack
+            int y = widgetsBox.top();
             if (displayAbove) {
-                  int mBoxHeight = mbox->size().height() + 15; // hack
-                  int y = widgetsBox.top();
+                  qDebug() << "Display Above";
```

Comment:
minor quibble: qDebug statements are generally fine, and useful when debugging your code, however a qDebug for something as trivial as this should not be included in a commit which gets put in the main codebase.

Reply:
Same here, please review